<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGetSingleHistoRunBlockDialog</class>
 <widget class="QDialog" name="PGetSingleHistoRunBlockDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get Single Histogram Run Block Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../musredit.qrc">
    <normaloff>:/images/musrfit.xpm</normaloff>:/images/musrfit.xpm</iconset>
  </property>
  <widget class="QGroupBox" name="fRunHeaderInfo_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Run Header Info</string>
   </property>
   <widget class="QLabel" name="fRunFileName_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Run File Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="fBeamline_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Beamline</string>
    </property>
   </widget>
   <widget class="QLabel" name="fInstitute_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Institute</string>
    </property>
   </widget>
   <widget class="QLabel" name="fFileFormat_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Format</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fRunFileName_lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>26</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fBeamline_lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>56</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="fInstitute_comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>86</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PSI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRIUMF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPARC</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="fFileFormat_comboBox">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>86</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NeXuS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ROOT-NPP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ROOT-PPC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSI-BIN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSI-MDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MUD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MDU-ASCII</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASCII</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DB</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fRequiredEntries_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Required Entries</string>
   </property>
   <widget class="QLabel" name="fMap_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="fForwardHistoNo_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward Histo No</string>
    </property>
   </widget>
   <widget class="QLabel" name="fDataRange_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data  Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="fBackgroundFix_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Fix</string>
    </property>
   </widget>
   <widget class="QLabel" name="fOr_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OR</string>
    </property>
   </widget>
   <widget class="QLabel" name="fBackgroundRange_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="fFitRange_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit Range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fMap_lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>26</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0  0  0  0  0  0  0  0  0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fForwardHistoNo_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>56</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fDataStart_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>116</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fDataEnd_lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>116</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fBackgroundFix_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>146</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fBackgroundStart_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>206</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fBackgroundEnd_lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>206</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fFitRangeStart_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>236</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fFitRangeEnd_lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>236</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fPacking_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>266</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="fPacking_textLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Packing/Binning</string>
    </property>
   </widget>
   <widget class="QLabel" name="fNorm_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Norm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fNorm_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>86</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fBackgroundFit_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Fit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fBackgroundFit_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>176</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fOptionalEntries_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Optional Entries</string>
   </property>
   <widget class="QLabel" name="fT0_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>t0 (reqired if not given in the data file)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fT0_lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>26</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fMuonLifetimeParameter_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Muon Lifetime Parameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fLifetime_lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>56</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="fLifetimeCorrection_checkBox">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>90</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lifetime Correction</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>545</y>
     <width>421</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="fHelp_pushButton">
        <property name="text">
         <string>&amp;Help</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>188</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fOk_pushButton">
        <property name="text">
         <string>&amp;OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fCancel_pushButton">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../musredit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fCancel_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetSingleHistoRunBlockDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOk_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetSingleHistoRunBlockDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fHelp_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetSingleHistoRunBlockDialog</receiver>
   <slot>helpContent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>helpContent()</slot>
 </slots>
</ui>
